trade.lic

```diff
@@ -193,9 +202,18 @@ class Trade
   end
 
   def start_up # should parse contracts, check mats, handle expireds, get the caravan to a useful outpost, deliver any contracts, then begin the loop.
-    get_caravan unless caravan_exists?
-    fput('look')
+    caravan_exists?
```

Comment:
❔ What is this line doing?

Reply:
Absolutely nothing anymore.  Good catch.  It also alerted me to a failure if someone tried to do the 'direct to room or outpost travel' without a caravan.